features/keychain/module/memoized-keychain.js

```diff
@@ -15,15 +14,15 @@ class MemoizedKeychain extends Keychain {
   #publicKeys = Object.create(null)
   #cloneOpts
 
-  constructor({ storage, logger }) {
-    super({ id: MODULE_ID, logger })
+  constructor({ storage, legacyPrivToPub }) {
+    super({ legacyPrivToPub })
 
```

Comment:
`MemoizedKeychain` extends `Keychain` but did not pass through `legacyPrivToPub` to the parent constructor, can't be right